src/main/java/frc/swerve/Wheel.java

```diff
@@ -50,9 +50,11 @@ public void enable() {
 		set(0, 0);
 	}
 
-	public void zero() {
+	/** Zeroes the wheel by updating the offset, and returns the new offset */
+	public int zero() {
 		System.out.println("Zeroing " + name + "module");
 		OFFSET = rotateMotor.getSelectedSensorPosition(0);
```

Comment:
Typically names in all capitals are for constants. Consider renaming this to `offset`.